### PR TITLE
Fix lonelynode creation

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by toabctl** [comment]
FYI: This should fix https://ci.suse.de/job/openstack-mkcloud/15363/consoleFull

**Comment by toabctl** [comment]
travis failure seems to be unrelated...

File: scripts/mkcloud [hunk 1]
```diff
@@ -580,9 +580,9 @@ function setuplonelynodes()
         local lonely_node
         lonely_node=$cloud-node$i
         ${mkcloud_lib_dir}/libvirt/compute-config $cloud $i $mac 0\
-            "$cephvolume" "$drbdvolume" $compute_node_memory\
+            "$cephvolumenumber" "$drbdvolume" $compute_node_memory\
             $controller_node_memory $libvirt_type $vcpus $emulator $vdisk_dir\
-            1 1 > /tmp/$cloud-node$i.xml
+            1 1 > /tmp/$cloud-node$i.xml || exit 5
```
Comment (MaximilianMeister) [inline scripts/mkcloud]: can you use a `complain` here instead of `exit`?

Comment (toabctl) [inline scripts/mkcloud]: ```complain``` is ```qa_crowbarsetup.sh``` only.

Comment (MaximilianMeister) [inline scripts/mkcloud]: `qa_crowbarsetup.sh` get's sourced in mkcloud, so it should be available

**Comment by aspiers** [comment]
Unrelated to this PR, but it would be great to replace the "lonely" naming with something more intuitive and precise like "preinstalled" (although probably someone else can think of a better word).

**Comment by toabctl** [comment]
@aspiers I agree. But that's a topic for another PR...

**Comment by dirkmueller** [comment]
+1

File: scripts/mkcloud (hunk 1 above)
Comment (jdsn) [inline scripts/mkcloud]: why not use `safely $cmd` ?

Comment (toabctl) [inline scripts/mkcloud]: @jdsn @MaximilianMeister  fixed in https://github.com/toabctl/automation/commit/9aac40f0b15aafb5519e78172084f909f175cb17

**Comment by jdsn** [comment]
:+1:

**Comment by jdsn** [comment]
the gating result does not make sense in this case, as the lonelynode setup is not covered by it

@toabctl do you want to rebuild this gating job with manually changed mkcloud target - or did you verify this fix manually?

**Pull request merged**